modules/index.php

```diff
@@ -5,10 +5,10 @@
  *
  * NOTICE OF LICENSE
  *
- * This source file is subject to the Open Software License (OSL 3.0)
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
```

Comment:
I have no idea if this change is legit 🤔

Reply:
I don't think so, this should use the Open source project license